edge-functions/svelte-middleware/src/routes/blocked.svelte

```diff
@@ -0,0 +1,10 @@
+<script context="module">
+  // disable client-side router to let requests be intercepted by Middleware
+  export const router = false
+</script>
```

Comment:
Can we maybe come up with a different example that doesn't require this — something like rewriting text or swapping out an image? IMHO it's a bit confusing if someone has to disable SvelteKit features in order to see the middleware do something. Redirects would normally be handled within the framework